movielens/setup.py

```diff
@@ -16,8 +16,10 @@
 
 NAME = 'movielens'
 VERSION = '1.0'
-TENSORFLOW_TRANSFORM = 'tensorflow-transform==0.1.10'
+SIX='six==1.10.0'
+TENSORFLOW='tensorflow==1.4.0rc'
```

Comment:
Also as we have discussed on other forums having TF specified on the setup file can harm the dataflow performance by downgrading the optimized binary that's already present on the runners.

Reply:
So what version is currently running with TFT 0.1.10? Oh its 1.3 as per the table in TFT right?

Reply:
Right now is 1.3, so I guess, we'll be fine. Thanks.